tensor_comprehensions/pybinds/tclib.cc

```diff
@@ -459,7 +490,19 @@ PYBIND11_MODULE(tclib, m) {
           "unchecked_run",
           &TcExecutor::uncheckedRun,
           py::arg("inputs"),
-          py::arg("outputs") = py::tuple());
+          py::arg("outputs") = py::tuple())
+      .def(
+          "profile",
+          &TcExecutor::profile,
+          py::arg("inputs"),
+          py::arg("outputs") = py::tuple())
+      .def(
```

Comment:
I don't think increasing the API complexity in this case is worth it: in userland one can just call profile repeatedly and build one's own p0, p25, p50, p90